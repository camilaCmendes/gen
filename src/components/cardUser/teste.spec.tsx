import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import { CardUser } from ".";

describe("CardUser Component", () => {
  const mockProps = {
    id: "1",
    userAccess: "ADMIN",
    name: "Camila de Carvalho Mendes",
    role: "Desenvolvedora Frontend",
    team: "Desenvolvimento",
    email: "camila@email.com",
    password: "123456",
    handleClickedCardUser: jest.fn(),
  };

  it("should render correctly and match snapshot", () => {
    const { asFragment } = render(<CardUser {...mockProps} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
