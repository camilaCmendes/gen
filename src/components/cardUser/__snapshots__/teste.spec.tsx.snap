// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardUser Component should render correctly and match snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid-root MuiGrid-container css-9cyib4-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-gj1fbr-MuiGrid-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-1nu5ncz-MuiTypography-root"
      >
        Camila de Carvalho Mendes
      </p>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-gj1fbr-MuiGrid-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-18tk2zh-MuiTypography-root"
      >
        Desenvolvedora Frontend
      </p>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-4 css-gj1fbr-MuiGrid-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-18tk2zh-MuiTypography-root"
      >
        Desenvolvimento
      </p>
    </div>
  </div>
</DocumentFragment>
`;
